modules/ssh/templates/sshd_config.erb

```diff
@@ -0,0 +1,28 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/. 
+
+# Accept locale-related environment variables
+AcceptEnv LANG LC_CTYPE LC_NUMERIC LC_TIME LC_COLLATE LC_MONETARY LC_MESSAGES
+AcceptEnv LC_PAPER LC_NAME LC_ADDRESS LC_TELEPHONE LC_MEASUREMENT
+AcceptEnv LC_IDENTIFICATION LC_ALL LANGUAGE
+AcceptEnv XMODIFIERS
+
+LogLevel VERBOSE
+
+<% if @only_user_ssh -%>
```

Comment:
only_user_ssh is a node scope var in puppet again.  There will be no node scope vars in ronin puppet.  There will also be no root user account logins PERIOD.  Therefore, it should always be 'PermitRootLogin no'